examples/plot_simulate_evoked.py

```diff
@@ -53,26 +53,28 @@
 import matplotlib.pyplot as plt
 fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
 plot_dipole(dpls, ax=axes[0], layer='agg', show=False)
-net.spikes.plot_hist(ax=axes[1], spike_types=['evprox', 'evdist'])
+net.cell_response.plot_hist(ax=axes[1], spike_types=['evprox', 'evdist'])
 
 ###############################################################################
 # Also, we can plot the spikes and write them to txt files.
 # Note that we can use formatting syntax to specify the filename pattern
 # with which each trial will be written. To read spikes back in, we can use
 # wildcard expressions.
-net.spikes.plot()
+net.cell_response.plot()
```

Comment:
humm ... now it seems this should be `plot_spikes`. We need to make all this consistent, but I'm okay if you want to push it to another PR

Reply:
Renamed it to `plot_spikes_raster()`. Additionally changed `plot_hist()` to `plot_spikes_hist()` for consistency.